xpdacq/simulation.py

```diff
@@ -190,6 +180,7 @@ def insert_imgs(mds, fs, n, shape, save_dir=tempfile.mkdtemp()):
 
 # instantiate simulation objects
 db = build_pymongo_backed_broker()
-pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: 5})
+db.fs.register_handler('RWFS_NPY', be.ReaderWithFSHandler)
+pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: np.ones((5,5))}, fs=db.fs)
```

Comment:
If you want you can steal the image cycle logic from [xpdSim](https://github.com/xpdAcq/xpdSim/pull/13/files#diff-a2f91388ba37eebfc35980ace3307f21R15) and the use of a data directory to use real images.